spec.bs

```diff
@@ -771,7 +771,7 @@ To run <dfn>make a component string</dfn> given a [=constructor string parser=]
   1. Let |component start token| be the result of running [=get a safe token=] given |parser| and |parser|'s [=constructor string parser/component start=].
   1. Let |component start input index| be |component start token|'s [=token/index=].
   1. Let |new length| be |token|'s [=token/index=] &minus; |component start input index|.
-  1. Return the substring within |parser|'s [=constructor string parser/input=] starting at |component start input index| with |new length|.
+  1. Return the [=code point substring=] from index |component start input index| with length |new length| within |parser|'s [=constructor string parser/input=].
```

Comment:
I think you could collapse this with the previous line by using `[=code point substring by indices|code point substring=] from indices |component start input index| to |token|'s [=token/index=] inclusive within |parser|'s [=constructor string parser/input=]`.